app/views/versions/edit.html.erb

```diff
@@ -27,8 +27,7 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <% html_title l("label_roadmap_edit", name: @version.name) %>
-
-<h2><%= Version.model_name.human %></h2>
+<%= toolbar title: Version.model_name.human %>
```

Comment:
DAFUQ?

I know that you only replaced code, but what does ``Version.model_name.human` mean? I hope it is something more clever than `Version.model_name.humanize`...